packages/go_router/doc/configuration.md

```diff
@@ -0,0 +1,95 @@
+Create a GoRouter configuration by calling the [GoRouter][] constructor and
```

Comment:
Should we merge this with get-started?

Reply:
I added a link to the configuration docs, I'd like to keep this file short if possible.